src/btree/bt_io.c

```diff
@@ -115,21 +109,19 @@ __wt_bt_read(WT_SESSION_IMPL *session,
 		 * here after corruption happens.  If we're salvaging the file,
 		 * it's OK, otherwise it's really, really bad.
 		 */
-		if (ret != 0 ||
-		    result_len != dsk->mem_size - WT_BLOCK_COMPRESS_SKIP) {
+		if (ret != 0 || result_len != dsk->mem_size - WT_BLOCK_COMPRESS_SKIP) {
 			fail_msg = "block decompression failed";
 			goto corrupt;
 		}
 	} else
-		/*
-		 * If we uncompressed above, the page is in the correct buffer.
-		 * If we get here the data may be in the wrong buffer and the
-		 * buffer may be the wrong size.  If needed, get the page
-		 * into the destination buffer.
-		 */
-		if (ip != NULL)
-			WT_ERR(__wt_buf_set(
-			    session, buf, ip->data, dsk->mem_size));
+	    /*
+	     * If we uncompressed above, the page is in the correct buffer.
+	     * If we get here the data may be in the wrong buffer and the
+	     * buffer may be the wrong size.  If needed, get the page
+	     * into the destination buffer.
+	     */
+	    if (ip != NULL)
```

Comment:
This indentation is a known issue from the last PR. I can wrap the inner `if` statement with braces to solve this.

Reply:
Wrapping the else in braces seems like something we could live with.